dynawo/3rdParty/CMakeLists.txt

```diff
@@ -904,12 +901,21 @@ else()
   set(LIBZIP_HOME ${install_dir})
 endif()
 
-# if ( CXX11_ENABLED )                                                                              # New part for IIDM-2
+#-wip- if ( CXX11_ENABLED )                                                                              # New part for IIDM-2
+
+# Minimum required for powsybl that sets that rule
+cmake_minimum_required(VERSION 3.12)
```

Comment:
This call can't be placed here, it has to be at the very top of a top CMakeLists.txt

https://cmake.org/cmake/help/latest/command/cmake_minimum_required.html

Reply:
I have read that. There is already a cmake_minimum_requirement at the very top. Nothing warns about the fact that requirement may change while processing, in fact requirement changes while treating other CMakeLists.txt by example. Writing it here makes it possible to change only one part of the big CMakeLists.txt. I tried it and did not notice any trouble.

Reply:
cf my comment on the top cmake_minimum_required deletion